lib/helper/import.js

```diff
@@ -0,0 +1,100 @@
+'use babel';
+
+const atom = global.atom;
+const Path = require('path');
+const FileSystem = require('fs-plus');
+const Electron = require('electron');
+const Storage = require('./../helper/storage.js');
+const XMLParser = require('xml-js');
+const server_config = require('./../config/server-schema.json');
+const folder_config = require('./../config/folder-schema.json');
+
+class Import {
+  constructor() {
+    const self = this;
+  }
+
+  import(configuration_view) {
+    const self = this;
+    Electron.remote.dialog.showOpenDialog(null, { title: 'Select file to import', defaultPath: Electron.remote.app.getPath("desktop"), buttonLabel: 'Import', filters: [{ name: 'All Files', extensions: ['*'] }, { name: 'remote-ftp config file', extensions: ['ftpconfig'] }, { name: 'FileZilla site export file', extensions: ['xml'] }], properties: ['openFile', 'showHiddenFiles'] }, (filePaths, bookmarks) => {
+      if (filePaths) {
+        filePaths.forEach((filePath, index) => {
+          if (Path.extname(filePath).toLowerCase() == '.xml') {
+            FileSystem.readFile(filePath, function (err, data) {
+              var json = XMLParser.xml2js(data, { compact: true });
+              if (typeof json['FileZilla3'] !== 'undefined') {
+                self.importFileZilla3(json['FileZilla3']['Servers']['Folder'], null);
+                configuration_view.reload();
+              }
+            });
+          } else if (Path.basename(filePath).toLowerCase() == '.ftpconfig') {
+            FileSystem.readFile(filePath, function (err, data) {
+              var json = JSON.parse(data);
+              self.importFtpConfig(json);
+              configuration_view.reload();
+            });
+
+          }
+        });
+      }
+
+    });
+  }
+
+  importFileZilla3(data, parent) {
+    const self = this;
+    if (typeof data['Server'] !== 'undefined') {
+      if (!Array.isArray(data['Server'])) {
+        data['Server'] = [data['Server']];
+      }
+      data['Server'].forEach((server_data, index) => {
+        let newconfig = JSON.parse(JSON.stringify(server_config));
+        newconfig.name = server_data['Name']['_text'];
```

Comment:
Can format your code with CamelCase variable names instead of _ underscores? I know it's a matter of taste but for the uniformity of the source code it would be better ;)